spec/requests/api/schemas/json/category_create_request.json

```diff
@@ -39,9 +39,13 @@
     },
     "search_priority": {
       "type": "integer"
+    },
+    "form_template_ids": {
```

Comment:
Just a thought here but I'm wondering if we need to update the API schema here to include this attribute which would not mean much at this point. Perhaps we want to exclude it from the API response until there is a valid use case for it?

Reply:
Ah yes, good point. I'll keep this in mind and perhaps add it later on when there is more of a valid case for it.